### DISCUSSION TITLE
Xcode Support on airyxOS

### DISCUSSION THREAD

**Discussion by OmerFlame** [ravynsoft/ravynos#191] [Ideas]
I am not asking for support now (since the project is in relatively early stages), but your macOS app support makes me wonder: will that ever be a thing? Developing Xcode apps on airyxOS sounds like a dream come true: an open-source operating system that lets you develop iOS apps and actually compile and run them onto a real device.

I just found about this project yesterday and this seems absolutely crazy! I am seriously impressed by your efforts and how far you actually got.

(Again, let me reiterate: I am not asking or begging for support. This just makes me wonder.)

**Comment by jhay06** [comment]
I'm hoping with this too .
